Lab6.ipynb

```diff
@@ -0,0 +1,443 @@
+      "text/plain": [
+       "<Figure size 1296x720 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "noise = stats.poisson.rvs(mu=100, size=100000)\n",
+    "injection = np.linspace(0,501,100000)\n",
+    "\n",
+    "observed = noise + injection\n",
+    "\n",
+    "h = plt.hist2d(observed, injection, bins = (100,100), density=True)\n",
+    "plt.colorbar(h[3])\n",
+    "plt.ylabel(\"Injected Signal\", fontsize = \"20\")\n",
+    "plt.xlabel(\"Observed Signal\", fontsize = \"20\")\n",
+    "plt.title(\"Confidence intervals\", fontsize = \"26\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Problem 3b"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In order to check with the previous result, we sample from the background many times and add to each sampled value the mean of the signal we had in the previously (250)."
```

Comment:
Problem 3B:
- Here, I get slightly lost in your method. But, it appears you still got the same result as before, but is it because you followed very similar steps to question 2? I'm not sure. The final answer with a mean of 350 makes sense. But I'm not sure I understood the math and graph for it.
- What I did was followed Lecture 13's+Homework 5's method exactly and just sliced along a horizontal or vertical line in the 2D histogram. So you could slice horizontally along True Injected Signal = 250 and when you histogram the possible observed values, you'll get the same mean, but directly from the 2D histogram. For the reverse problem, you could slice vertically along Observed Signal = X and find the spread of possible Injected values. I think that was what we were supposed to do. But, since you did get the same answer, this might still be fine.